Sites/styles.css

```
@@ -0,0 +1,80 @@
+body {
+    @media (max width: 1280px) @media (min-width: 375px) height: 4600px;
+    background: linear-gradient(#ebf0fa, #adc2eb);
+}
+
+h1 {
+    font-family: Bodoni;
+    color: #0B243B;
+    font-weight: bold;
+    font-size: 50px;
+    text-align: center;
+}
+
+.container {
+    margin: auto;
+    max-width: 1280px;
+    min-width: 375px;
+}
+
+h2 {
+    font-family: 'Garamond';
```

Same note about including a font stack.
```suggestion
    font-family: Garamond, Baskerville, "Baskerville Old Face", "Hoefler Text", "Times New Roman", serif;
```